include/onnxruntime/core/framework/allocator.h

```diff
@@ -9,6 +9,15 @@
 #include "ortdevice.h"
 #include "ortmemoryinfo.h"
 
+// This configures the arena based allocator used by ORT
+// See docs/C_API.md for details on what these mean and how to choose these values
+struct OrtArenaCfg {
```

Comment:
Moved from the C API header file to an internal header and placed in the global namespace